examples/demo_actor/README.md

```diff
@@ -4,7 +4,7 @@ This document describes how to create an Actor(DemoActor) and invoke its methods
 
 * **The actor interface(demo_actor_interface.py).** This contains the interface definition for the actor. Actor interfaces can be defined with any name. The interface defines the actor contract that is shared by the actor implementation and the clients calling the actor. Because client may depend on it, it typically makes sense to define it in an assembly that is separate from the actor implementation.
 
-* **The actor service(demo_actor_service.py).** This implements Flask web service that is going to host the actor. It contains the implementation of the actor, `demo_actor.py`. An actor implementation is a class that derives from the base type `Actor` and implements the interfaces defined in `demo_actor_interface.py`.
```

Comment:
Add one point for the Flask example also?

Reply:
Sure I will add

Reply:
Added.